File: solutions/python/raindrops/1/raindrops.py
""" converts a number into pling plang Plong"""
def convert(number):
    """ converts a number into pling plang Plong"""
    result = ""
    if number % 3 == 0:
        result += "Pling"

    if number % 5 == 0:
        result += "Plang"

    if number % 7 == 0:
        result += "Plong"

    if result == "":
        result = str(number)

    return result

File: solutions/go/raindrops/1/raindrops.go
package raindrops

import "strconv"

// Convert - changes a number into raindrops
func Convert(n int) string {
    var rd string = ""
    if n % 3 == 0 { rd = rd + "Pling"}
    if n % 5 == 0 { rd = rd + "Plang"}
    if n % 7 == 0 { rd = rd + "Plong"}
    if rd == "" { rd = strconv.FormatInt(int64(n), 10)}
    return rd
}
